CSharp/ArticleProjects/TaskDependencies/Program.cs

```diff
@@ -58,77 +54,91 @@ public static void Main(string[] args)
+                        new CloudTask("Sun", "cmd.exe /c echo Sun"),
  
-                    // Tasks 1, 2, and 3 don't depend on any other tasks. Because
-                    // we will be using them for a task range dependency, we must
-                    // specify string representations of integers as their ids.
-                    new CloudTask("1", "cmd.exe /c echo 1"),
-                    new CloudTask("2", "cmd.exe /c echo 2"),
-                    new CloudTask("3", "cmd.exe /c echo 3"),
+                        // Task 'Flowers' depends on completion of both 'Rain' and 'Sun'
+                        // before it is run.
+                        new CloudTask("Flowers", "cmd.exe /c echo Flowers")
+                        {
+                            DependsOn = TaskDependencies.OnIds("Rain", "Sun")
+                        },
  
-                    // Task 4 depends on a range of tasks, 1 through 3
-                    new CloudTask("4", "cmd.exe /c echo 4")
-                    {
-                        // To use a range of tasks, their ids must be integer values.
-                        // Note that we pass integers as parameters to TaskIdRange,
-                        // but their ids (above) are string representations of the ids.
-                        DependsOn = TaskDependencies.OnIdRange(1, 3)
-                    },
-
-                    // Task 5 depends on a range of tasks, 1 through 3, and 'Flowers'
-                    new CloudTask("5", "cmd.exe /c echo 5")
-                    {
-                        DependsOn = new TaskDependencies(
-                            new[] { "Flowers" },
-                            new[] { new TaskIdRange(1, 3) })
-                    },
-                };
-
-                // Add the tasks to the job.
-                await batchClient.JobOperations.AddTaskAsync(jobId, tasks);
-
-                // Pause execution while we wait for the tasks to complete, and notify
-                // whether the tasks completed successfully.
-                Console.WriteLine("Waiting for task completion...");
-                Console.WriteLine();
-                CloudJob job = await batchClient.JobOperations.GetJobAsync(jobId);
+                        // Tasks 1, 2, and 3 don't depend on any other tasks. Because
+                        // we will be using them for a task range dependency, we must
+                        // specify string representations of integers as their ids.
+                        new CloudTask("1", "cmd.exe /c echo 1"),
+                        new CloudTask("2", "cmd.exe /c echo 2"),
+                        new CloudTask("3", "cmd.exe /c echo 3"),
+
+                        // Task A is the parent task.
+                        new CloudTask("A", "cmd.exe /c echo A")
+                        {
+                            // Specify exit conditions for task A and their dependency actions.
+                            ExitConditions = new ExitConditions()
```

Comment:
Do not use `()` in the parameterless constructor call when supplying an initialiser list.  (Same comment throughout this code block.)